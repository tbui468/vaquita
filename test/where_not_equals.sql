create database sol;
open sol;

create table planets (id int key, name string, moons int, rings bool);
insert into planets (id, name, moons, rings) values (1, "Mars", 2, true), (2, "Earth", 1, false);
select * from planets where name <> "Earth";
select * from planets where moons <> 1;
select * from planets where rings <> false;

close sol;
drop database sol;
exit;
