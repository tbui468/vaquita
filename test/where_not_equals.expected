created database sol
opened database sol
created table planets
inserted 2 record(s) into planets
id, name, moons, rings
1, Mars, 2, true
id, name, moons, rings
1, Mars, 2, true
id, name, moons, rings
1, Mars, 2, true
closed database sol
dropped database sol
